6_common_genes_analysis/1_common_DEGs_and_gene_groups.py

```diff
@@ -419,12 +441,10 @@
 
 # **Takeaway:**
 #
-# * Based on the venn diagrams, it looks like most common DEGs are core, as expected. Since it is thought that these core genes encode essential functions shared by all strains, it would make sense that these core genes are also those commonly DEGs.
-#
-# * Based on the Fisher's exact test results, there is an odds ratio <1 indicating that there is a negative relationship between a gene being common DEGs and a gene being core vs accessory. With the p-value indicating that this relationship is significant.
-#     * This [paper](https://www.d.umn.edu/~tpederse/Pubs/scsug96.pdf) talks about the p-values obtained from the Fisher's exact tests are reliable compared to asymptotic test results when dealing with skewed/unbalanced datasets. Furthermore, [this blog](https://courses.lumenlearning.com/boundless-statistics/chapter/the-chi-squared-test/) suggests that while a $\chi^2$ test is recommended for large datasets, like what we have, in the case where the dataset is skewed/unbalanced the p-values for the Fisher's exact test are more reliable.
+# Here our null hypothesis is that common DEGs are equally likely to be core or accessory genes.
+# In order to calculate the significance of the observed data, i.e. the total probability of observing data as extreme or more extreme if the null hypothesis is true using the Fisher's exact test.
+# We found that the percentage of PAO1 core genes that are common is ~25% compared to ~34% accessory genes. In PA14, the percentage of core genes that are common is ~25% compared to ~25% accessory genes.
 #
-# * For the Fisher’s exact test, the null hypothesis is that common DEGs are equally likely to be core or accessory genes.
-#     * In order to calculate the significance of the observed data, i.e. the total probability of observing data as extreme or more extreme if the null hypothesis is true
-#     * Percentage of core genes that are common is ~19-20% compared to ~26-32% accessory genes
-#     * There is a negative relationship, meaning if a gene is common DEG, its less likely it’s a core gene
+# Overall, in PAO1 common genes are more likely to be accessory. The genes commonly differentially expressed in response to these stress conditions, are changes that are commonly needed for _P. aeruginosa_ to establish itself within the host.
```

Comment:
Do you think this is tied to what you saw in `3_core_core_analysis/5_KEGG_enrichment_of_stable_genes.ipynb`, where the least stable core genes were also (very weakly) enriched for stress response genes? Maybe less stable core genes almost act like accessory genes in some ways?

Just throwing ideas out there, you've probably thought about this much more than I have.

Reply:
Yea I that's our guess right now but I still need to think about this some more!